apis/python/setup.py

```diff
@@ -162,9 +167,9 @@ def run(self):
 
 
 INC_DIRS = [
-    "../../libtiledbsoma/include",
-    "../../libtiledbsoma/external/include",
-    "../../build/externals/install/include",
+    "dist_links/libtiledbsoma/include",
+    "dist_links/libtiledbsoma/external/include",
```

Comment:
`dist_links/libtiledbsoma` is just a symlink to `../../libtiledbsoma` when we're in the git source tree. When we're in an extracted sdist, it's a materialized copy of that directory (resulting from pip following the symlink as it created the sdist tarball). There's other stuff above where we're conditionally using `dist_links/libtiledbsoma` or `../../libtiledbsoma` that can probably be changed to use the symlink unconditionally -- I think that may have been my original oversight.